### DISCUSSION TITLE
Kibana not starting after upgrade to 2.3.14

### DISCUSSION THREAD

**Discussion by oneCrazyAdmin** [Security-Onion-Solutions/securityonion#8389] [General] [answered]
Good Afternoon,

I am getting a 404 page when trying to browse to /kibana/

I looked in the kibana error logs and I am getting this message

{"ecs":{"version":"8.0.0"},"@timestamp":"2022-07-27T23:29:02.104+00:00","message":"[.kibana_task_manager] Action failed with 'security_exception: [security_exception] Reason: action [indices:admin/block/add] is unauthorized for user [anonymous_user] with roles [superuser] on restricted indices [.kibana_task_manager_7.17.4_001], this action is granted by the index privileges [manage,all]'. Retrying attempt 11 in 64 seconds.","log":{"level":"ERROR","logger":"savedobjects-service"},"process":{"pid":7},"trace":{"id":"9c9fc0828635e0f09455b926a90911b9"},"transaction":{"id":"d02b4bb219d6412f"}}

Has anyone come across this?

**Comment by dougburks** [comment, marked as answer]
Did you follow all steps at https://docs.securityonion.net/en/2.3/soup.html#elastic-8 including the following?
> If your deployment was originally installed with a version lower than 2.3.60 and you never enabled Elastic Auth, then you will need to enable Elastic Auth via [so-elastic-auth](https://docs.securityonion.net/en/2.3/so-elastic-auth.html#so-elastic-auth) before updating.

**Reply by oneCrazyAdmin** [reply]
I did not do that.  I thought it was a recommendation, not a requirement.  Can I roll back to a previous version and make the change?

**Reply by dougburks** [reply]
You can try enabling Elastic Auth without rolling back but you may need to restart some services.

**Reply by oneCrazyAdmin** [reply]
Thank you for pointing me in the right direction.  I used so-elastic-auth to enable it (true), then disable it (false).

Then I went to the docshttps://docs.securityonion.net/en/2.3/soup.html like you said.

There were these two commands that I ran that seemed to help as well
`sudo rm /opt/so/state/kibana_*.txt

sudo salt-call state.apply kibana.so_savedobjects_defaults -l info queue=True`

I am all back up and running now!
